2001-3000/LIT2892RepCh173.xml

```diff
@@ -23,11 +23,6 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <sourceDesc>
             <p>Work of the literatures of Ethiopia and Eritrea</p>
          </sourceDesc>
-         <sourceDesc>
-            <listWit>
-               <witness corresp="BNFabb88">B.A. 88</witness>
-            </listWit>
-         </sourceDesc>
       </fileDesc>
```

Comment:
please put the witness back since you do use it in the record. just without the text

```
<sourceDesc>
            <listWit>
               <witness corresp="BNFabb88"/>
            </listWit>
         </sourceDesc>
```

Reply:
Done! Thank you!

Reply:
If you have corrected the file you must have forgotton to commit your changes to the branch as they are not visible yet